run_tests.py

```diff
@@ -24,6 +25,8 @@
 
     # Silence too verbose Django logging
     logging.disable(logging.DEBUG)
+    # TODO: Delete this after upgrading `xblock-sdk`.
+    warnings.simplefilter(action='ignore', category=FutureWarning)
```

Comment:
The newer version of `xblock-sdk` was throwing these warnings after every single test, making them unreadable.